docs/build/spells.md

```diff
@@ -0,0 +1,128 @@
+# Spells
+
+Spells are actors in the Fluence Network designed to perform tasks with user business logic as well as maintenance jobs. 
+
+Think of them as of automated jobs, that can be executed when some specific trigger event occurs (time, connection poll, http, blockchain events, etc).
+
+:::info
+Older implementation of Fluence had Scheduled Scripts, stateless AIR scripts which were periodically run by the node. There had rather restricted use cases since:
+- they were stateless
+- they could be triggered only by a timer
+- they were working on behalf of the node
+- they couldn’t communicate with each other
+- they didn’t have a good management interface
+
+Spells are basically Scheduled Scripts done ~~right~~ better.
+:::
+
+Spells can be used for user-defined automation scenarios for service maintenance, periodical data extraction, subnet maintenance, etc. 
+
+A spell is a marine service, the main goal of which is to provide state storage for script execution. It also provides some additional features to ease the spell usage for users and take some responsibility from the node (aka Nox). The service serves as a template and is instantiated on the creation of a new spell.
+
+This service is intended to be used as a template and instantiated by a node on each new spell creation.
```

Comment:
It feels like we need to reformulate these. I don't think this concept is understandable to outsiders.